internal/attestation/gcp/issuer.go

```diff
@@ -26,11 +26,12 @@ type Issuer struct {
 
 // NewIssuer initializes a new GCP Issuer.
 func NewIssuer() *Issuer {
+	GetInstanceInfo := getGCEInstanceInfo(metadataClient{})
 	return &Issuer{
 		Issuer: vtpm.NewIssuer(
 			vtpm.OpenVTPM,
 			tpmclient.GceAttestationKeyRSA,
-			getGCEInstanceInfo(metadataClient{}),
+			GetInstanceInfo,
```

Comment:
Why this change?

Reply:
IMO instantiating the function implies you plan on using it more than once, which is not the case here.
Additionally private variables and functions should start with a lowercase character.

Reply:
I agree, it might imply wrong usage, I will change it back as it was.